bind_test.go

```diff
@@ -91,35 +91,35 @@ var bindTestCases = [...]xmpptest.FeatureTestCase{
 		State:      xmpp.Received,
 		Feature:    xmpp.BindCustom(bindFunc),
 		In:         `<iq type="set" id="123" to="example.net" from="test@example.net"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"><resource>test</resource></bind></iq>`,
-		Out:        `<iq xmlns="jabber:server" type="result" to="test@example.net" from="example.net" id="123"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"><jid>test@example.net/test</jid></bind></iq>`,
+		Out:        `<iq xmlns="jabber:client" type="result" to="test@example.net" from="example.net" id="123"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"><jid>test@example.net/test</jid></bind></iq>`,
```

Comment:
To have a regression test, let's change one of these to using `jabber:server`, then in the `State` field above set it to `State:xmpp.Received|xmpp.S2S` and the test should pass and use the correct namespace.

Reply:
I've amended my commit, but when running the test I was required to add the server namespace to the "In" IQ and I'm not sure if it's normal

Reply:
Yah, that should be fine.